std/v2alpha/range/primitives.d

```diff
@@ -0,0 +1,33 @@
+module std.v2alpha.range.primitives;
+
+/**
+@@@TODO@@@ Publicly imported names across modules should copy or link those
+names' respective documentation. Here, we reuse empty and isInputRange because
+we don't change their meaning in v2.
+*/
+public import std.range.primitives : empty, isInputRange;
+
+/**
+@@@TODO@@@ This function redefines `front` for v2, meaning its documentation
+will override the documentation of `front` found in v1. The difference is of
+course that the new `front` does not autodecode.
```

Comment:
Adding a link to the v1 documentation makes it easier for people to see what's been updated.